src/constants/model.js

```diff
@@ -95,7 +99,9 @@ module.exports = {
     YETI: 'Yeti'
   },
   [Make.TOYOTA]: {
-    HIACE: 'Hiace'
+    HIACE: 'Hiace',
+    AURIS: 'Auris',
+    HI_LUX: 'Hi-Lux'
```

Comment:
```suggestion
    HILUX: 'Hilux'
```

https://en.wikipedia.org/wiki/Toyota_Hilux